content/INTRODUCTION.md

```diff
@@ -0,0 +1,18 @@
+**Requirements**: No programming experience, at least 12 years old
+
+**Duration**: 
+
+**Standards**: K-12 Computer Science Framework Core Concepts and Practices
+* Algorithms and Programming
+  - Collaborating Around Computing
+  - Recognizing and Defining Computational Problems
+  - Developing and Using Abstractions
+  - Creating Computational Artifacts
+  - Testing and Refining Computational Artifacts
+
+## Web Development Curriculum
+
+Learn to code and start building professional, career-boosting websites and web applications.
+
+The `Web Development Curriculum` is designed to develop the logical, analytical, programming, and web development skills of individuals who want to become full-stack web developers. In this curriculum, you will learn comprehensive concepts and principles on how to create dynamic websites and web applications using several tools and technologies such as `HTML`, `CSS`, `Javascript`, `DOM`, `PHP`, and `PostgreSQL`. You will also experience to solve and work on real-world projects and problems encountered by professional web developers.
+
```

Comment:
Place `how` between `experience` and `to`

`You will also experience how to solve and work on real-world projects and problems encountered by professional web developers.`

Let me know what you think. Thanks.

Reply:
I think adding `how` is not appropriate to what I want to inform to the readers. I will try to rephrase this statement. Thanks.